packages/wdio-browserstack-service/src/insights-handler.ts

```diff
@@ -71,35 +95,179 @@ class _InsightsHandler {
         }
     }
 
-    async beforeHook (test: Frameworks.Test, context: any) {
+    getCucumberHookType(test: CucumberHook|undefined) {
+        let hookType = null
+        if (!test) {
+            hookType = this._cucumberData.scenariosStarted ? 'AFTER_ALL' : 'BEFORE_ALL'
+        } else if (!this._cucumberData.stepsStarted) {
+            hookType = 'BEFORE_EACH'
+        } else if (this._cucumberData.steps?.length > 0) {
+            // beforeStep or afterStep
```

Comment:
did we intend to do something here?

Reply:
Yes, but for future when we starts handling step hooks for cucumber